<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_project_wizard_page_1</class>
 <widget class="QWizardPage" name="new_project_wizard_page_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Project File Order</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="fileListWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="removeListButton">
          <property name="text">
           <string>Remove Selected</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Clip Area Polygon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="clipLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="clipOpenFileButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Text File List</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="txtOpenFileButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="txtFileToListButton">
          <property name="text">
           <string>Add to List</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QToolButton" name="importFileButton">
            <property name="text">
             <string>Import Existing HDF</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="downloadDataButton">
            <property name="text">
             <string>Download New Data</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>EPSG code</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="epsgLineEdit">
              <property name="text">
               <string>3413</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Pixel Size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pixelSizeLineEdit">
              <property name="text">
               <string>250</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>HDF Bands (comma separated)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="hdfBandsLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
